app/poros/cast_member.rb

```diff
@@ -0,0 +1,26 @@
+class CastMember
+  attr_reader  :name,
+               :character,
+               :order,
+               :picture,
+               :gender,
+               :id
+  def initialize(member)
+    @name = member[:name]
+    @character = member[:character]
+    @order = member[:order]
+    @picture = member[:profile_path]
+    @gender = format_gender(member[:gender])
+    @id = member[:id]
+  end
+
+  def format_gender(data)
```

Comment:
I love that you are forward-thinking. Not all actors might have their gender classified as simply male or female as we move into our new world!